README.md

```diff
@@ -1,51 +1,100 @@
-## This contract will keep track of votes for different proposal on near blockchain
+# `Voting Contract` 
 
-Right now it only has two proposal and it does not allow any aditional proposal so you can vote either for
+This is an example smart contract on the [NEAR protocol](https://near.org/developers/). This example keeps track of votes for a proposal that you can deploy on the NEAR blockchain. These are the steps to deploy to the testnet.
 
-- NFT MAKER APP
-- HomePage
+## Instructions
 
-### Contract Name
+The example is intended for developers and some level of techincal expertise is assumed. If you don't know what terminal is, this probably isn't for you.
```

Comment:
I would remove this part _If you don't know what terminal is, this probably isn't for you._  just to be more friendly :)

Reply:
haha yes good idea to be friendly the first paragraph :)

i removed this line